Documentation/project-docs/porting.md

```diff
@@ -0,0 +1,113 @@
+# Porting to .NET Core
+
+The purpose of this document is to share the plans on how we're going to port more APIs to .NET Core.
+
+## Summary
+
+Compared to other .NET stacks, specifically the .NET Framework, Mono, Unity, and Xamarin the .NET Core platform doesn't expose enough APIs. This makes porting assets to .NET Core quite challenging, especially for areas which don't have replacements.
+
+So we want to bring more API to .NET Core. In order to keep our promise of being open and transparent, we want to:
+
+* Clearly communicate what we plan on porting, how we prioritize, what we don't want to port, and how we're thinking of cross-platform and compatibility.
+* Avoid being the bottleneck by allowing community members to help with porting.
+
+## Prioritization
+
+As of today, .NET Core supports three app models:
+
+* **ASP.NET apps and services**
+* **Console apps**
+* **Universal Windows Platform (UWP) apps**
+
+We'll prioritize APIs that affect the ability to write those kind of apps over app models that .NET Core currently doesn't support (such as building desktop applications). Please note that this doesn't mean that we'll never port the other APIs -- it just means those rank lower.
```

Comment:
Non-related, but "... So we want to bring more API to .NET Core...", I think "API" should be in plural.